relevanceai/_api/endpoints/api_client.py

```diff
@@ -83,9 +83,7 @@ def _convert_id_to_string(self, documents, create_id: bool = False):
             )
         except KeyError:
             if create_id:
-                self.set_field_across_documents(
-                    "_id", [make_id(document) for document in documents], documents
-                )
+                pass
```

Comment:
Why was this removed? 
If this was because of the ID hashing, we should instead switch over to UUID.

Reply:
the API handles ID creation if you dont set any.